content/docs/components/metadata.md

```diff
@@ -4,9 +4,6 @@ description = "Tracking and managing metadata of machine learning workflows in K
 weight = 5
 +++
 
-{{% beta-status 
```

Comment:
Is the metadata component now considered **stable** for Kubeflow v1.0? We need to mark each component with an alpha, beta, or stable banner, as defined in the [versioning policy](https://master.kubeflow.org/docs/reference/version-policy/). I think we should retain the beta-status banner here.

Reply:
Beta banner is perfect. Thanks @sarahmaddox.  I added it back.

I was looking at the v0.7 doc and thinking it is alpha banner.

Reply:
Thanks!